modules/vaos/README.rdoc

```diff
@@ -0,0 +1,10 @@
+= Vaos
+Rails engine module for the VAOS (VAMF) integration.
+
+== Installation
+Ensure the following line is in the root project's Gemfile:
+
+  gem 'vaos', path: 'modules/vaos'
+
+== License
+This module is open source under the terms of the [MIT License](https://opensource.org/licenses/MIT).
```

Comment:
vets-api has a license [CC0 1.0 Universal Summary](https://github.com/department-of-veterans-affairs/vets-api/blob/master/LICENSE.md) and I don't think we want a separate license for gems.